tests/e2e/csi_snapshot_basic.go

```diff
@@ -562,17 +624,25 @@ var _ = ginkgo.Describe("[block-vanilla-snapshot] Volume Snapshot Basic Test", f
 		12. Delete volume snapshot content 2
 		13. Cleanup the pvc, volume snapshot class and storage class
 	*/
-	ginkgo.It("Verify snapshot static provisioning with deletion policy Retain", func() {
+	ginkgo.It("[tkg-snapshot] Verify snapshot static provisioning with deletion policy Retain", func() {
```

Comment:
Is this test applicable to tkg now?

Reply:
No, this testcase also requires a new code writing. 
I am reverting the changes back to its original code.